code/__HELPERS/roundend.dm

```diff
@@ -199,6 +199,10 @@
 	if(length(CONFIG_GET(keyed_list/cross_server)))
 		send_news_report()
 
+	//tell the nice people on discord what went on before the salt cannon happens.
+	world.TgsTargetedChatBroadcast("The current round has ended. Please standby for your shift interlude Nanotrasen News Network's report!", FALSE)
+	world.TgsTargetedChatBroadcast(send_news_report(),FALSE)
```

Comment:
TGS doesn't automatically account for rate limiting when outputting to Discord, and Discord's rate limit is pretty strict last I checked. Might be a good idea to use only one targetedchatbroadcast here

Reply:
Maybe. It'd make the message pretty messy tho.

Reply:
p sure you can pass a linebreak through it somehow

Reply:
I'll see how it does first, then just make it a blob of msg += s